src/style.css

```diff
@@ -0,0 +1,87 @@
+body {
+  margin-top: 0;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.searchBar {
+  width: 100%;
+  background: #fff;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  font-family: "Avenir", sans-serif;
+}
+
+input::placeholder {
+  color: #3d4e61;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
```

Comment:
Input will broke if u add some element to top of the form, let's add relative to the input wrapper

Reply:
done